### PR TITLE
BB-42 run oob locally

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

File: docs/run-oob-locally.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+```
+bin/vaultclient create-account --name account1 --email dev@null1 --port 8600
+bin/vaultclient generate-account-access-key --name account1 --port 8600
+aws configure --profile account1
+```
+## AWS S3 CLI
+
+Create a bucket into this account with ingestion enabled location:
+```
+aws s3api create-bucket --bucket bucket1 --create-bucket-configuration LocationConstraint=wontwork-location:ingest --endpoint-url http://127.0.0.1:8000 --profile account1
```
Comment (miniscruff) [inline docs/run-oob-locally.md]:
```suggestion
aws s3api create-bucket \
    --bucket bucket1 \
    --create-bucket-configuration LocationConstraint=wontwork-location:ingest \
    --endpoint-url http://127.0.0.1:8000 \
    --profile account1
```

Having to scroll is tedious

File: docs/run-oob-locally.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+## CloudServer
+
+/!\ IMPORTANT: locationConfig.json should be identical to the backbeat conf/locationConfig.json.
+Add Ring access key and secret key in locationConfig.json:
+https://github.com/scality/backbeat/blob/1657782020009d63be8f3df342917305dea7671b/conf/locationConfig.json
+
+```
+REMOTE_MANAGEMENT_DISABLE=true S3VAULT=multiple yarn start_mongo
```
Comment (miniscruff) [inline docs/run-oob-locally.md:89]: I am not able to get multiple to work and had to use mem, not sure if I did something wrong but I just get a generic bad healthcheck startup thingy.

File: docs/run-oob-locally.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+aws s3api list-objects --endpoint-url http://217.182.11.36 --profile s3consoleaccount --bucket ring-bucket-1
+```
+
+## Zenko-client
+
+Resume ingestion for location “wontwork-location”
+Note: by default locations ingestion state is set to paused.
+```
+git checkout feature/ARTESCA-1849/poc
+node oob.js
```
Comment (miniscruff) [inline docs/run-oob-locally.md:177]: Where is this file? I don't see it anywhere.

File: docs/run-oob-locally.md (hunk 2 above)
Comment (nicolas2bert) [reply docs/run-oob-locally.md:89]: I did not faced this issue with `development/8.2` branch. We can look at it.

File: docs/run-oob-locally.md (hunk 3 above)
Comment (nicolas2bert) [reply docs/run-oob-locally.md:177]: https://github.com/scality/zenkoclient/blob/feature/ARTESCA-1849/poc/oob.js

File: docs/run-oob-locally.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+git checkout feature/ARTESCA-1849/poc
+```
+Add Ring access key and secret key in conf/locationConfig.json:
+```
+"accessKey": "TO_BE_ADDED",
+"secretKey": "TO_BE_ADDED"
+```
+Start backbeat API:
+```
+MANAGEMENT_BACKEND=operator REMOTE_MANAGEMENT_DISABLE=true yarn run start
```
Comment (miniscruff) [inline docs/run-oob-locally.md]:
```suggestion
MANAGEMENT_BACKEND=operator REMOTE_MANAGEMENT_DISABLE=true yarn start
```

File: docs/run-oob-locally.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+## Backbeat - Ingestion producer
+```
+MANAGEMENT_BACKEND=operator yarn ingestion_populator
+```
+## Backbeat - Ingestion processor
+```
+MANAGEMENT_BACKEND=operator yarn run mongo_queue_processor
```
Comment (miniscruff) [inline docs/run-oob-locally.md]:
```suggestion
MANAGEMENT_BACKEND=operator yarn mongo_queue_processor
```

File: docs/run-oob-locally.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+Create the necessary data directories for each member by issuing a command similar to the following:
+
+```
+mkdir -p /tmp/mongodb/rs0-0 /tmp/mongodb/rs0-1 /tmp/mongodb/rs0-2
+```
+
+Start your mongod instances in their own shell windows by issuing the following commands:
+```
+mongod --replSet rs0 --port 27018 --bind_ip localhost --dbpath /tmp/mongodb/rs0-0 --smallfiles --oplogSize 128
```
Comment (miniscruff) [inline docs/run-oob-locally.md]: I got an error using `--smallfiles` but could be my mongo version as I have 4.4

**Review by miniscruff — COMMENTED** [review]
Was able to get it to work eventually

File: docs/run-oob-locally.md (hunk 6 above)
Comment (nicolas2bert) [reply docs/run-oob-locally.md]: oh might be I was using v3.6.20

Comment (naren-rajendran) [reply docs/run-oob-locally.md]: I think we should mimic production as much as we can..so match the version from production.

File: docs/run-oob-locally.md (hunk 4 above)
Comment (naren-rajendran) [reply docs/run-oob-locally.md]: `yarn run <name>` is the full syntax though? does the short form matter?

File: docs/run-oob-locally.md (hunk 3 above)
Comment (naren-rajendran) [reply docs/run-oob-locally.md:177]: @miniscruff did you get it to work?

Comment (miniscruff) [reply docs/run-oob-locally.md:177]: Yea I did, just had to clone the zenko client

Comment (miniscruff) [reply docs/run-oob-locally.md:177]: For what it is worth though I did have to edit the script cause I use different names of things

**Review by naren-rajendran — APPROVED** [review]
Looks good to me.

**Review by miniscruff — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**